jupyterhub_carina/CarinaOAuthClient.py

```diff
@@ -33,8 +33,9 @@ class CarinaOAuthClient(LoggingConfigurable):
     CARINA_OAUTH_HOST = os.environ.get('CARINA_OAUTH_HOST') or 'oauth.getcarina.com'
     CARINA_AUTHORIZE_URL = "https://%s/oauth/authorize" % CARINA_OAUTH_HOST
     CARINA_TOKEN_URL = "https://%s/oauth/token" % CARINA_OAUTH_HOST
-    CARINA_PROFILE_URL = "https://%s/me" % CARINA_OAUTH_HOST
-    CARINA_CLUSTERS_URL = "https://%s/clusters" % CARINA_OAUTH_HOST
+    CARINA_PROFILE_URL = "https://%s/users/current" % CARINA_OAUTH_HOST
+    CARINA_CLUSTERS_URL = "https://%s/proxy/clusters" % CARINA_OAUTH_HOST
+    CARINA_TEMPLATES_URL = "https://%s/proxy/cluster_types" % CARINA_OAUTH_HOST
```

Comment:
Nice, these are looking like good paths for Carina now. (I'm supposed to be reviewing your PR, I can't help but ponder about Carina itself :wink).